breadth_first_search/bfs.py

```diff
@@ -0,0 +1,43 @@
+edge={}
+
+def addEdges(source, destination):
+    if(source not in edge):
+        edge[source]=[]
+    edge[source].append(destination)
+
+def breadthFirstSearch(source, destination):
+    bfs=[source]
+    queue=[source]
+    visited=[source]
+    flag=0
+    
+    while(len(queue)!=0):
+        source=queue.pop(0)
+        temp=[]
+        if(source in edge and len(edge[source])>0):
+            temp=list(edge[source])
+        for i in temp:
+            if(i not in visited):
+               bfs.append(i)
+               if(i==destination):
+                   flag=1
+                   break
+               queue.append(i)
+               visited.append(i)
+
+        if(flag==1):
+            break;
+    if(flag==0):
+        return
+    return bfs
+    

```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpttqwh6ut/breadth_first_search/bfs.py
+++ b/tmp/tmpttqwh6ut/breadth_first_search/bfs.py
@@ -30,7 +30,7 @@
     if(flag==0):
         return
     return bfs
-    
+
 addEdges("A", "B");
 addEdges("A", "D");
 addEdges("B", "C");
```